### PR TITLE
CLOUDP-124937: Openshift image certification

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

Added a script to certify the operator image for the Openshift

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).

### CODE REVIEW

**Review by Sugar-pack — APPROVED** [review]
LGTM

File: .github/actions/certify-openshift-images/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest
+
+COPY entrypoint.sh /home/entrypoint.sh
+RUN chmod +x /home/entrypoint.sh
+# Code file to execute when the docker container starts up (`entrypoint.sh`)
+ENTRYPOINT ["/home/entrypoint.sh"]
```
Comment (fabritsius) [inline .github/actions/certify-openshift-images/Dockerfile:6]: A few files have "no new line" thing, probably better to fix it 😅

**Review by fabritsius — APPROVED** [review]
Great Job 👍 
Left just one small suggestion :)

File: .github/actions/certify-openshift-images/Dockerfile (hunk 1 above)
Comment (fabritsius) [reply .github/actions/certify-openshift-images/Dockerfile:6]: This symbol is not shown here but it is present in the [file changes](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/646/files).

File: .github/actions/certify-openshift-images/action.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: 'certify-openshift-images'
+description: 'push image to RedHat Connect for certification'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```
Comment (igor-karpukhin) [inline .github/actions/certify-openshift-images/action.yaml:5]: image: 'Dockerfile'
```suggestion
  image: 'Dockerfile'

```

File: .github/workflows/release-certify.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+      - name: Certify image and create release for Certified Operators repo
+        uses: ./.github/actions/certify-openshift-images
+        with:
+          IMAGE: quay.io/mongodb/mongodb-atlas-kubernetes-operator
+          VERSION: ${{ steps.tag.output.version }}
+          RH_CERTIFIED_OPENSHIFT_REPO_PATH: ../certified-operators
+          RH_CERTIFICATION_OSPID: ${{ secrets.RH_CERTIFICATION_OSPID }}
+          RH_CERTIFICATION_TOKEN: ${{ secrets.RH_CERTIFICATION_TOKEN }}
+          RH_CERTIFICATION_PYXIS_API_TOKEN: ${{ secrets.RH_CERTIFICATION_PYXIS_API_TOKEN }}
```
Comment (igor-karpukhin) [inline .github/workflows/release-certify.yaml:63]:
```suggestion
          RH_CERTIFICATION_PYXIS_API_TOKEN: ${{ secrets.RH_CERTIFICATION_PYXIS_API_TOKEN }}

```

File: scripts/release-redhat-certified.sh [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+# Add skip range
+value='">=0.8.0"' yq e -i '.spec.skipRange = env(value)' \
+  "${REPO}/${VERSION}"/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
+
+cd "${REPO}"
+git checkout -b "mongodb-atlas-kubernetes-operator-${VERSION}"
+git add "${REPO}/${VERSION}"
+git commit -m "operator mongodb-atlas-kubernetes (${VERSION})" --signoff
+git push mongodb "mongodb-atlas-operator-community-${VERSION}"
+cd -
```
Comment (igor-karpukhin) [inline scripts/release-redhat-certified.sh:84]:
```suggestion
cd -

```

**Review by fabritsius — APPROVED** [review]

**Pull request merged**